src/components/PressableWithSecondaryInteraction/index.ios.js

```diff
@@ -10,25 +10,28 @@ import {propTypes, defaultProps} from './pressableWithSecondaryInteractionPropTy
  * @param {Object} props
  * @returns {React.Component}
  */
-const PressableWithSecondaryInteraction = props => (
-    <Pressable
-        ref={props.forwardedRef}
-        onPress={props.onPress}
-        onPressIn={props.onPressIn}
-        onLongPress={(e) => {
-            e.preventDefault();
-            ReactNativeHapticFeedback.trigger('selection', {
-                enableVibrateFallback: true,
-            });
-            props.onSecondaryInteraction(e);
-        }}
-        onPressOut={props.onPressOut}
-        // eslint-disable-next-line react/jsx-props-no-spreading
-        {...(_.omit(props, 'onLongPress'))}
-    >
-        {props.children}
-    </Pressable>
-);
+const PressableWithSecondaryInteraction = (props) => {
+    const Node = props.inline ? RNText : Pressable;
```

Comment:
@parasharrajat I think it might be worth adding a little comment explaining this option here as well. I know you have added the comment to the PropType, but one compact line in here would be handy. Can you please add one? Thank you 🙌